src/TurbulenceClosures/turbulence_closure_implementations/smagorinsky_lilly.jl

```diff
@@ -60,7 +60,7 @@ Lilly, D. K. "On the numerical simulation of buoyant convection." Tellus (1962)
 Smagorinsky, J. "General circulation experiments with the primitive equations: I.
     The basic experiment." Monthly weather review (1963)
 """
-SmagorinskyLilly(FT=Float64; C=0.23, Cb=1.0, Pr=1.0, ν=0, κ=0,
+SmagorinskyLilly(FT=Float64; C=0.16, Cb=1.0, Pr=1.0, ν=0, κ=0,
```

Comment:
```suggestion
SmagorinskyLilly(FT=Float64; C=0.17, Cb=1.0, Pr=1.0, ν=0, κ=0,
```

Can we add a citation for the constant?

Reply:
I believe 0.165 is what Lilly finds, but I'll look it up. There's a reference for that in the docs though (it says Lilly1966).

Reply:
Hmm so I guess it depends if you want to round up or down 🤔

Reply:
Probably we want to round down since it's most likely to be used for shear flows (in which case it might need to be even smaller than 0.16...)

Reply:
I think the reference in the docs is indeed the first time it appears. Although I could only find a pre-publication review copy online. If you do that math the actual value is around 0.165. Some people round up (like Pressel), but most truncate it to 0.16 (as it is in the docs), so I'll keep using that value and include a ref to it in the docstring.